synapse/lib/storm.py

```diff
@@ -1084,6 +1084,53 @@ def reqValidPkgdef(pkgdef):
         }
         ''',
     },
+    {
+        'name': 'nodes.import',
+        'descr': 'Import a nodes file hosted at a URL into the cortex. Yields created nodes.',
+        'cmdargs': (
+            ('urls', {'nargs': '*', 'help': 'URL(s) to fetch nodes file from'}),
+            ('--no-ssl-verify', {'default': False, 'action': 'store_true', 'help': 'Ignore SSL certificate validation errors.'}),
+        ),
+        'storm': '''
+        init {
+            $count = (0)
+            function fetchnodes(url, ssl) {
+                $resp = $lib.inet.http.get($url, ssl_verify=$ssl)
+                if ($resp.code = 200) {
+                    $nodes = $lib.list()
+                    for $valu in $resp.msgpack() {
+                        $nodes.append($valu)
+                    }
+                    yield $lib.feed.genr("syn.nodes", $nodes)
+                } else {
+                    $lib.exit("nodes.import got HTTP error code: {code} for {url}", code=$resp.code, url=$url)
```

Comment:
This is probably the only point of discussion; since this could teardown a pipeline of multiple inet:url nodes if one is missing.

Reply:
So my main question for that would be, if there's a lot of inet:url nodes passing lifted and passed into this, is `$lib.warn` sufficient to let the user know that something has failed? I opted for the more strict case, but that was mostly me guessing.